test.txt

```diff
@@ -0,0 +1 @@
+Test fil - repræsenterer kode der addresserer en bug
```

Comment:
The changes look ok, but please write all comments in english as we are a global company with developers all over the world